test/unit/option_builders_test.rb

```diff
@@ -123,9 +123,13 @@ def resource_names(builders)
     end
 
     it "adds dependent searchable option builders" do
-      resource_names(builders_by_class(HammerCLIForeman::DependentSearchablesOptionBuilder)).must_equal [
-        :compute_resources
-      ]
+      resources = resource_names(builders_by_class(HammerCLIForeman::DependentSearchablesOptionBuilder)).sort
+      if FOREMAN_VERSION < Gem::Version.new('1.10')
```

Comment:
Same as in https://github.com/theforeman/hammer-cli-foreman/pull/203/files#r46411880